Sources/Apodini/Components/Handler/HandlerIdentifier.swift

```diff
@@ -8,35 +8,66 @@
 
 import Foundation
 
+private struct AllAnyHandlerIdentifiersAnchor: TruthAnchor {}
 
-/// An `AnyHandlerIdentifier` object identifies a `Handler` regardless of its concrete type.
-open class AnyHandlerIdentifier: Codable, RawRepresentable, Hashable, Equatable, CustomStringConvertible, KnowledgeSource {
-    public let rawValue: String
+/// A `KnowledgeSource` that initializes and holds all `AnyHandlerIdenfier`, by also asserting the uniqueness of them.
+struct AllAnyHandlerIdentifiers: KnowledgeSource {
```

Comment:
Just a suggestion, but why didn’t you make it private? I don’t see a reals use case where you would want to access all identifiers (especially on a global (not exporter specific) scope.